src/Root_TEST.cc

```diff
@@ -670,3 +670,33 @@ TEST(DOMRoot, WorldByName)
   ASSERT_TRUE(root.WorldNameExists("world2"));
   EXPECT_EQ("world2", root.WorldByName("world2")->Name());
 }
+
+/////////////////////////////////////////////////
+TEST(DOMRoot, ClearActorLightModel)
+{
+  sdf::Root root;
+  EXPECT_EQ(nullptr, root.Actor());
+  EXPECT_EQ(nullptr, root.Light());
+  EXPECT_EQ(nullptr, root.Model());
+
+  sdf::Actor actor1;
+  actor1.SetName("actor1");
+  root.SetActor(actor1);
+  EXPECT_NE(nullptr, root.Actor());
```

Comment:
```suggestion
  EXPECT_NE(nullptr, root.Actor());
  EXPECT_EQ(nullptr, root.Light());
  EXPECT_EQ(nullptr, root.Model());
```
let's check all three pointers, expecting at most one to be non-null